### PR TITLE
Add Docs for new Azure Storage Integration

### PR DESCRIPTION
**Pull request by DanielLanger**
Currently in Beta, so the docs have a beta tag.

Reason for publishing is folks are asking :)

### CODE REVIEW

File: content/integrations/azure_storage.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+integration_title: Microsoft Azure Storage
+kind: integration
+git_integration_title: azure_storage
+newhlevel: true
+beta: true
+---
+
+# Overview
+
+This feature is currently in Beta. To enable it for your account, please reach out to <support@datadoghq.com>
```
Comment (kmshultz) [inline content/integrations/azure_storage.md]: I would finish the Overview with this rather than lead in.

Comment (kmshultz) [inline content/integrations/azure_storage.md]: Actually, the `beta: true` tag causes a red-background BETA warning to be displayed below the title and above the Overview. However it does not say 'Please reach out at support@datadog to enable it', so we could keep that part here. I would still add it at the end, though.

**Comment by DanielLanger** [comment]
@kmshultz done

**Comment by irabinovitch** [comment]
:+1:

**Pull request merged**